src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -20,6 +20,17 @@ public TagHelperAttribute()
         }
 
         /// <summary>
+        /// Instantiates a new instance of <see cref="TagHelperAttribute" /> with values provided by the given
+        /// <paramref name="attribute"/>.
+        /// </summary>
+        /// <param name="attribute">A <see cref="IReadOnlyTagHelperAttribute"/> whose values should be copied.</param>
+        public TagHelperAttribute(IReadOnlyTagHelperAttribute attribute)
+            : this (attribute.Name, attribute.Value)
```

Comment:
`(attribute?.Name, attribute?.Value)` just because the `NotNull` check won't happen prior to the `this` call.